graphstorm-processing/docker/0.2.2/emr-serverless/Dockerfile.cpu

```diff
@@ -63,7 +63,8 @@ RUN python -m pip install --no-deps /usr/lib/spark/code/graphstorm_processing-*.
     rm /usr/lib/spark/code/graphstorm_processing-*.whl && rm -rf /root/.cache
 
 FROM runtime AS test
-RUN python -m pip install --no-deps /usr/lib/spark/code/graphstorm-processing/ && rm -rf /root/.cache
+RUN python -m pip install --no-deps /usr/lib/spark/code/graphstorm-processing/ mock && \
```

Comment:
Do we need to install pytest here? I check our pyproject.toml does not include pytest package here.

Reply:
Testing in the SageMaker image at least it seems like the env already has pytest. We can rely on that for now, or have an explicit optional dependency group for it (like we currently have the `dev` dependency group).